### PR TITLE
Release 1.17.2

### PR DESCRIPTION
**Pull request by vapurrmaid**
- [x] Generate template changelog from script
- [x] Fill in Changelog
- [x] Merge all feature documentation
- [x] Technical Editor Reviews
- [ ] Documentation Owner Review (Will review tomorrow: Mar 31 2021)
- [x] Release Manager Review

### CODE REVIEW

**Review by sreya — APPROVED** [review]

File: changelog/1.17.2.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    embeddable quickstart button.
+  - web: New options available when clicking **New Environment** from the
+    **Environments** page
+- web: Added _Cordon_ and _Uncordon_ actions to _Provider_ audit logs
+
+### Bug Fixes 🐛
+
+- infra: Fixes an issue whereby using `coder/configure` to create Dev URLs would
+  fail
+- infra: Fixes an issue whereby authenticating with GitLab OIDC may fail if name
```
Comment (jawnsy) [inline changelog/1.17.2.md:25]: The problem is that GitLab's OIDC does not return a name at all, so I would suggest phrasing as

```suggestion
- infra: Fixes an issue authenticating using OpenID Connect Identity Providers (IdPs) that omit name information, such as GitLab. In this case, Coder will use the email address as the user's name.
```

**Review by jawnsy — APPROVED** [review]

File: environments/workspaces-as-code/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Workspaces as code (WAC) brings the _Infrastructure As Code_ paradigm to Coder
+environments. WAC allows you to define and create new environments using
+**workspace templates**.
+
+[Workspace Templates](./templates.md) are declarative YAML files that describe
+how to configure environments and their supporting infrastructure.
+
+## Requirements
+
+- You must configure a [GitHub or GitLab OAuth service](../../admin/git.md)
```
Comment (mterhar) [inline environments/workspaces-as-code/index.md:16]: Does this not work with bitbucket?

File: environments/workspaces-as-code/templates.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    - key: my-key
+      operator: equals
+      value: my-value
+      effect: NoExecute
+      tolerationSeconds: 3600
+```
+
+#### workspace.spec.gpucount
+
+The number of GPUs to inject into the environment.
```
Comment (mterhar) [inline environments/workspaces-as-code/templates.md:122]: I think inject is the wrong term here but may be wrong. I'd say "allocate" if it were my sentence.

File: environments/workspaces-as-code/index.md (hunk 2 above)
Comment (sreya) [reply environments/workspaces-as-code/index.md:16]: We'll have it for 1.18

**Pull request merged**